### PR TITLE
HTTP Reporter added

### PR DESCRIPTION
**Pull request by ghostec**

### CODE REVIEW

File: reporters/http.go [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+// HTTP reports metrics to an http endpoint
+type HTTP struct {
+	client handlers.Client
+	region string
+	logger *logrus.Logger
+}
+
+// HTTPClient implements reporters::http.Client interface
+type HTTPClient struct {
+	client *http.Client
```
[the remark this answers is not in the document]
Comment (cscatolini) [reply reporters/http.go]: ❤️

[the remark this answers is not in the document]
Comment (ghostec) [reply reporters/http.go]: Great! Done.

File: reporters/http.go [hunk 2]
```diff
@@ -103,14 +106,16 @@ func MakeHTTP(config *viper.Viper, logger *logrus.Logger, r *Reporters) {
...
+	c.SetDefault("reporters.http.timeoutMs", "5000")
 }
 
 // NewHTTP creates an HTTP struct using putURL and region from config
 func NewHTTP(config *viper.Viper, logger *logrus.Logger) (*HTTP, error) {
 	loadDefaultHTTPConfigs(config)
 	putURL := config.GetString("reporters.http.putURL")
 	region := config.GetString("reporters.http.region")
-	client := NewHTTPClient(putURL)
+	timeoutMs := config.GetInt64("reporters.http.timeoutMs")
```
Comment (Henrod) [inline reporters/http.go]: on config/local.yaml you should save, for example, as `5s` and use here `config.GetDuration`. So no need to convert to time.Duration in line 118

File: reporters/http_test.go [hunk 3]
```diff
@@ -83,7 +84,7 @@ var _ = Describe("HTTP", func() {
 					w.WriteHeader(http.StatusCreated)
 				}),
 			)
-			client := reporters.NewHTTPClient(ts.URL)
+			client := reporters.NewHTTPClient(ts.URL, time.Duration(5000)*time.Millisecond)
```
Comment (Henrod) [inline reporters/http_test.go]: `5000 * time.Millisecond` works

Comment (ghostec) [reply reporters/http_test.go]: Done!

**Pull request merged**